Makefile

```diff
@@ -0,0 +1,23 @@
+build:
+	go build ./...
+
+test:
+	go test ./...
+
+hello:
+	curl localhost:8080/api/v1/resource/my-resource
+
+mlmd-proto:
+	rm -rf /tmp/ml-metadata && \
+	git clone https://github.com/google/ml-metadata.git /tmp/ml-metadata && \
```

Comment:
Should we tag a particular version of MLMD? It would be easier when upgrading.

Reply:
Add a tag env.